WordPress/src/main/java/org/wordpress/android/ui/reader/discover/interests/ReaderInterestsFragment.kt

```diff
@@ -80,18 +86,28 @@ class ReaderInterestsFragment : Fragment(R.layout.reader_interests_fragment_layo
             updateDoneButton(uiState.doneButtonUiState)
             with(uiHelpers) {
                 updateVisibility(progress_bar, uiState.progressBarVisible)
-                updateVisibility(title, uiState.titleVisible)
+                updateVisibility(title, if (entryPoint == EntryPoint.DISCOVER) uiState.titleVisible else false)
```

Comment:
❓: Can we extract this title visibility if condition to view model?

Reply:
👋 @ashiagr !

You are totally right, on it.

Reply:
⚠️ @ashiagr !

This is now done: 9955d140007ec9a6cd529a7f95e6e7df082eac2e and 2db3918ea140b2dfb34ef41cf611812c90dbc348